src/patternfly/components/Button/button.scss

```diff
@@ -553,6 +559,12 @@
     color: var(--#{$button}--m-plain--Color);
     background-color: var(--#{$button}--m-plain--BackgroundColor);
 
+    &.pf-m-inline {
+      padding: var(--#{$button}--m-plain--m-inline--PaddingTop) var(--#{$button}--m-plain--m-inline--PaddingRight) var(--#{$button}--m-plain--m-inline--PaddingBottom) var(--#{$button}--m-plain--m-inline--PaddingLeft);
```

Comment:
Do you want to go ahead and make this logical properties?

Reply:
Do we plan on updating CSS vars similarly at any point, e.g. instead of `--component--PaddingTop` => `--component--PaddingBlockStart` or something along those lines? Probably makes more sense to keep the `PaddingTop` etc var names to be less confusing, but curious if that's on the roadmap in regards to RTL.

Reply:
@coker and I talked about that - we were thinking to keep the variables as is because that would be a sweeping breaking change.